PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -97,15 +169,14 @@ struct HfFragmentationFunctionTask {
     registry.add("h_d0_phi", ";#phi_{D^{0}} (GeV/c^{2});dN/d#phi_{D^{0}}", {HistType::kTH1F, {{250, -10., 10.}}});
   }
 
-  void processDummy(aod::TracksIU const&) {}
-  PROCESS_SWITCH(HfFragmentationFunctionTask, processDummy, "Dummy process function turned on by default", false);
+  void processDummy(aod::TracksIU const& tracks) {}
```

Comment:
you should remove the tracks variable since its unused

Reply:
All right, will remove